hydra-main/src/main/java/com/addthis/hydra/query/aggregate/QueryTask.java

```diff
@@ -145,7 +182,74 @@ private static AutoField getSourceField(Query query) {
                 throw new UncheckedIOException(e);
             }
         } else {
-            return new NoopField();
+            return null;
```

Comment:
that said, not objecting to the change, just please use the appropriate annotations.